09-Function_operators.Rmd

```diff
@@ -478,8 +478,8 @@
     And then we build two versions. One via an anonymous function and one via `partial()`:
     
     ```{r, eval}
-    compose_red <- function(fs) {
-      Reduce(function(f, g) function(...) f(g(...)), fs)
+    compose_red <- function(...) {
+      Reduce(function(f, g) function(...) f(g(...)), list(...))
     }
     compose_red(c(mean, length, unique))(1:10)
```

Comment:
I think you now have to change it to `compose_red(mean, length, unique)`. ...The previous version also could compose any number of functions, they just had to be passed as an explicit vector.

Reply:
P.S. Thanks for the contribution! 👍